models/repo_list_test.go

```diff
@@ -51,84 +56,110 @@ func TestSearchRepositoryByName(t *testing.T) {
 	assert.Equal(t, int64(1), count)
 
 	repos, count, err = SearchRepositoryByName(&SearchRepoOptions{
-		Keyword:  "test_repo",
-		Page:     1,
-		PageSize: 10,
-		Private:  true,
+		Keyword:     "test_repo",
+		Page:        1,
+		PageSize:    10,
+		Private:     true,
+		Collaborate: util.OptionalBoolFalse,
 	})
 
 	assert.NoError(t, err)
 	assert.Equal(t, int64(3), count)
 	assert.Len(t, repos, 3)
 
+	// Test non existing owner
+	repos, count, err = SearchRepositoryByName(&SearchRepoOptions{OwnerID: int64(99999)})
```

Comment:
nit: use `NonexistentID`

Reply:
`SearchRepositoryByName` doesn't check if user exists. This is checked on higher levels (eg api handler).

Reply:
OK, got it...